week3/61-sistiema-linieinykh-uravnienii-1/task.py

```diff
@@ -0,0 +1,25 @@
+def safe_to_int(n):
+    if n.is_integer():
+        return int(n)
+    else:
+        return n
+
+
+def main():
+    a, b, c = float(input()), float(input()), float(input())
+    d, e, f = float(input()), float(input()), float(input())
+    # a_11 * x + a_12 * y = b_1
+    # a_21 * x + a_22 * y = b_2
+    # (from task)
+    # a_11 is not zero,because system has exactly one solution
```

Comment:
we chose non null x coefficient as a_11. such non null coefficient exists, because system has exactly one solution